ruby/neo4j/driver/ext/config_converter.rb

```diff
@@ -31,6 +31,8 @@ def config_method(key, value)
           when 'resolver'
             proc = value
             value = ->(address) { java.util.HashSet.new(proc.call(address)) }
+          when 'trust_strategy'
+            value = Java::OrgNeo4jDriver::Config::TrustStrategy.trust_all_certificates
```

Comment:
so this does not match the above. If you replace a class like TrustStrategy you have to be able to replace all its usages across the entire code base. That's why it is not a good idea to start with simple terminal (leaf) classes but rather with classes that are only used from already converted ruby code. There is a bit of those and with each conversion more are added to this set.
In other words, the ruby code and the still-active java code together should at no point reference both ruby and java version of a the same class.

Reply:
Yup, I agree, making changes to not leave out same class in both ruby-java 👍